Nirajan total sums fix

n7koirala opened the pull request.

hamishun reviewed (approved):
Docs. Test. Seems all good.

faberga reviewed (approved):
Review + Travis build&test passed

n7koirala committed:
Fix totalSums

- Fixed totalSums function for non-power of 2 slot-size
- Added logic to increase noise bound for correct decryption

n7koirala committed:
Update EncryptedArray.cpp

n7koirala committed:
Fix totalSums

- Fixed totalSums function for accumulating ciphertext slots with slot size as a non-power of 2 
- Increased the noise bound during accumulation for correct decryption

n7koirala committed:
Fixed totalSums

Made totalSums work correctly for non-power of 2 slot-size
